scenarios/dashboard.json

```diff
@@ -1,7 +1,8 @@
 {
   "scenarios": [{
     "label": "Dashboard",
-    "url": "{url}/civicrm/dashboard"
+    "url": "{url}/civicrm/dashboard",
+    "onReadyScript": "dashboard/main.js"
```

Comment:
`main` doesn't really say what the script does. Would `show.js` be better?

Reply:
yes! that will be better.